### PR TITLE
Add math logic to calculator

### PR DESCRIPTION
**Pull request by AmineHLub**
<h1> Summary </h1>

- Add math logic using given files calculate.js & operate.js
- Add BIG dependency
- Remove manifest from html

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,56 +1,50 @@
 import React, { Component } from 'react';
 import '../App.css';
+import calculate from '../logic/calculate';
 
 export default class Calculator extends Component {
```
Comment (Georjane) [inline src/components/Calculator.js:5]: - [ ] Your calculator works perfectly well but can you please implement the use of life cycle methods as this was one of the learning objectives of this project

**Review by Georjane — CHANGES_REQUESTED** [review]
### Hello, @AmineHLub, ✋🏽 

Good job so far! You have a very professional Readme file, your PR is professional and you have a good mastery of git-flow. Your linters are all passing, I applaud that too.
However, we have a few more changes to make in order to meet all requirements for this project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding! 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 🤝 

_Code Reviewer
**[Jane](https://witahgeorjane.tech/)**_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/Calculator.js (hunk 1 above)
Comment (AmineHLub) [reply src/components/Calculator.js:5]: Hello, @Georjane I did read about implementing life cycle. And I believe there is no need for implementing it yet. My project is one component only and there is nothing much, no API call no async functions yet.. I tried and Implemented an update method to show the current operation (using previous state snapshot). I can use its return like this 
image

but that would not look like the given UI. Should I go ahead with it ?

Comment (Georjane) [reply src/components/Calculator.js:5]: Hi @AmineHLub, please which life cycle method did you use to implement this exactly?

Comment (AmineHLub) [reply src/components/Calculator.js:5]: Hey @Georjane . Thank you very much for responding quickly it's appreciated. I used `componentDidUpdate`. exactly as 

```
  componentDidUpdate(prevProps, prevState) {
    const fullOperation = () => {
      if (prevState.total && prevState.operation && prevState.next) {
        return (
          <p className="full-operation">
            {prevState.total}
            {' '}
            {prevState.operation}
            {' '}
            {prevState.next}
            {' '}
          </p>
        );
      }
      return null;
    };
    return (
      fullOperation()
    );
  }
```

Comment (Georjane) [reply src/components/Calculator.js:5]: Great! This is looks perfect! I think the learning objectives here have all been attained.👍💯

Comment (AmineHLub) [reply src/components/Calculator.js:5]: @Georjane I hope you can know how much your comment relieved me :relieved: . Thank you very much I will push the updates and ask for another code review

**Review by Emmyn5600 — APPROVED** [review]
Hi, @AmineHLub 

Good job so far! 👏🏼

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous reviewer.

 #### Optional suggestions
 
 - N/A
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**